src/open_inwoner/accounts/tests/test_action_views.py

```diff
@@ -327,10 +327,19 @@ def setUp(self) -> None:
 
     def test_action_status(self):
         context = self.browser.new_context(storage_state=self.user_login_state)
+        self.cookiebanner_enabled = True
 
         page = context.new_page()
         page.goto(self.live_reverse("profile:action_list"))
 
+        # find the decline button inside the cookie-banner
+        cookiebanner_banner = page.locator(".cookie-banner")
+        cookiebanner_rejectbutton = page.locator(".reject-button")
+        # close cookie-banner
+        cookiebanner_rejectbutton.click()
+        # check cookie-banner state
+        expect(cookiebanner_banner).to_be_hidden()
+
         # find action element and the dropdown widget
```

Comment:
1. The test is for the action view, which has nothing specifically to do with cookies. This would need to be a separate test.
2. If we're going to test the hiding of the cookie banner, we should also test that it's hidden when the 'accept' button is clicked.

However, I'm not sure this is even needed. We can confirm manually that the banner is hidden on click. What we do need to check automatically is that the analytics scripts are not rendered when the cookie is not set. You should modify the test in `src/open_inwoner/configurations/tests/test_analytics.py` (I should have done this myself when I changed the test, but didn't think of it back then).